widget/entry.go

```diff
@@ -898,6 +898,38 @@ func (e *Entry) registerShortcut() {
 	e.shortcut.AddShortcut(&fyne.ShortcutSelectAll{}, func(se fyne.Shortcut) {
 		e.selectAll()
 	})
+
+	moveWord := func(s fyne.Shortcut) {
+		row := e.textProvider().row(e.CursorRow)
+		start, end := getTextWhitespaceRegion(row, e.CursorColumn, true)
+		if start == -1 || end == -1 {
+			return
+		}
+
+		e.setFieldsAndRefresh(func() {
+			if s.(*desktop.CustomShortcut).KeyName == fyne.KeyLeft {
+				if e.CursorColumn == 0 {
+					if e.CursorRow > 0 {
```

Comment:
I suppose these could use a single if-statement.

Reply:
I don't think so - the outer if has an else, the inner is a condition of the first `if`.

Reply:
But it has the same effect, has it not? If we have `e.CursorColumn == 0 && e.CursorRow > 0` and the first part of the and operation fails, we will fall into the else statement.

Reply:
I don't see why this would be wrong. Avoids the triple if statements and instead keeps it down to two.
```go
if e.CursorColumn == 0 && e.CursorRow > 0 {

} else {
	
}
```

Reply:
What about the case where `e.CursorColumn = 0` and `e.CursorRow = 0`?
In the original code we do nothing, in your revised code it calls to else, which it shouldn't (as far as I can see)...